components/molecules/AuthSection/auth-section.tsx

```diff
@@ -42,6 +43,12 @@ const AuthSection: React.FC = ({}) => {
     }
   }, []);
 
+  useEffect(() => {
```

Comment:
Jsut wondering if in a future PR we can handle the redirect server side?

Reply:
Yep. I chose not to mix it in with the workspaces work for now